docs-partials/builder/vsphere/common/HardwareConfig-not-required.mdx

```diff
@@ -25,7 +25,10 @@
 - `displays` (int32) - Number of video displays. See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-789C3913-1053-4850-A0F0-E29C3D32B6DA.html)
   for supported maximums. Defaults to 1.
 
-- `vgpu_profile` (string) - vGPU profile for accelerated graphics. See [NVIDIA GRID vGPU documentation](https://docs.nvidia.com/grid/latest/grid-vgpu-user-guide/index.html#configure-vmware-vsphere-vm-with-vgpu)
+- `pci_passthrough_allowed_device` ([]PCIPassthroughAllowedDevice) - Configure Dynamic DirectPath I/O PCI Passthrough for virtual machine. See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-5B3CAB26-5D06-4A99-92A0-3A04C69CE64B.html)
```

Comment:
The documentation references the PCIPassThroughAllowedDevice type but does not link to it. I recommend adding a link to the documentation https://developer.hashicorp.com/packer/plugins/builders/vmware/iso#vmware-iso-builder-configuration-reference and including the partial inside in the vsphere-iso builder docs.

Reply:
I've addressed this in https://github.com/hashicorp/packer-plugin-vsphere/commit/c53c318d61d2d4cfb87c136e5927be73e90feaad.